Fix destination area highlighting

oliverroick opened the pull request:
This change addresses two things:

- Updates the destination-highlight when the map is zoomed
- Limit the destination-highlight to the top 5 areas by transportation volume

Changes added:

- Order the destination area data by flow value
- Move querying the feature IDs to the `applyDestinationAreaIdsToMap` action. That way we can query the feature IDs using the current map view
- Add a `zoomend` event that triggers the `applyDestinationAreaIdsToMap`

vgeorge commented on package.json:
```diff
@@ -78,5 +78,6 @@
     }
-  }
+  },
+  "packageManager": "pnpm@9.15.4+sha512.b2dc20e2fc72b3e18848459b37359a32064663e5627a51e4c74b2c29dd8e8e0491483c3abb40789cfd578bf362fb6ba8261b05f0387d76792ed6e23ea3b1b6a0"
```
@oliverroick why do we need this? I had to remove this line in order to run locally.

oliverroick replied:
I was on the wrong node version this is added automatically when  you run `pnpm install` in newer node versions that use corepack.

We can prevent this by pinning the required node version to 20 and add engine-strict=true. That way it will throw and error when you use a different node version. What do you think?

vgeorge replied:
@oliverroick I'm used to running `nvm i` when I switch to the project, but it makes sense to pin the version in package.json. But maybe as an enhancement in a separated PR?

vgeorge reviewed (approved):
@oliverroick I visited different areas and they all seem to render area highlighting correctly. Other that the pnpm issue in the package.json, this looks good to merge.

oliverroick commented:
@vgeorge I've added a couple more commits here

- Walk back the top-five highlight as discussed in the meeting
- Added colours to the bar chart
- Display the name of the selected area in a popup

Can you review again?

vgeorge commented:
@oliverroick this looks nice! I can see the color and the marker. Other than the `pnpm` issue, this looks good to merge.
